ios/CodePush/SSZipArchive/aes/pwd2key.c

```diff
@@ -101,93 +101,3 @@ void derive_key(const unsigned char pwd[],  /* the PASSWORD     */
             key[k++] = ux[j++];
     }
 }
-
-#ifdef TEST
-
-#include <stdio.h>
-
-struct
-{   unsigned int    pwd_len;
-    unsigned int    salt_len;
-    unsigned int    it_count;
-    unsigned char   *pwd;
-    unsigned char   salt[32];
-    unsigned char   key[32];
-} tests[] =
-{
-    {   8, 4, 5, (unsigned char*)"password",
-        {   
-            0x12, 0x34, 0x56, 0x78 
-        },
-        {   
-            0x5c, 0x75, 0xce, 0xf0, 0x1a, 0x96, 0x0d, 0xf7,
-            0x4c, 0xb6, 0xb4, 0x9b, 0x9e, 0x38, 0xe6, 0xb5 
-        }
-    },
-    {   8, 8, 5, (unsigned char*)"password",
-        {   
-            0x12, 0x34, 0x56, 0x78, 0x78, 0x56, 0x34, 0x12 
-        },
-        {   
-            0xd1, 0xda, 0xa7, 0x86, 0x15, 0xf2, 0x87, 0xe6,
-            0xa1, 0xc8, 0xb1, 0x20, 0xd7, 0x06, 0x2a, 0x49 
-        }
-    },
-    {   8, 21, 1, (unsigned char*)"password",
-        {
-            "ATHENA.MIT.EDUraeburn"
-        },
-        {
-            0xcd, 0xed, 0xb5, 0x28, 0x1b, 0xb2, 0xf8, 0x01,
-            0x56, 0x5a, 0x11, 0x22, 0xb2, 0x56, 0x35, 0x15
-        }
-    },
-    {   8, 21, 2, (unsigned char*)"password",
-        {
-            "ATHENA.MIT.EDUraeburn"
-        },
-        {
-            0x01, 0xdb, 0xee, 0x7f, 0x4a, 0x9e, 0x24, 0x3e, 
-            0x98, 0x8b, 0x62, 0xc7, 0x3c, 0xda, 0x93, 0x5d
-        }
-    },
-    {   8, 21, 1200, (unsigned char*)"password",
-        {
-            "ATHENA.MIT.EDUraeburn"
-        },
-        {
-            0x5c, 0x08, 0xeb, 0x61, 0xfd, 0xf7, 0x1e, 0x4e, 
-            0x4e, 0xc3, 0xcf, 0x6b, 0xa1, 0xf5, 0x51, 0x2b
-        }
-    }
-};
-
-int main()
-{   unsigned int    i, j, key_len = 256;
-    unsigned char   key[256];
-
-    printf("\nTest of RFC2898 Password Based Key Derivation");
-    for(i = 0; i < 5; ++i)
-    {
-        derive_key(tests[i].pwd, tests[i].pwd_len, tests[i].salt,
-                    tests[i].salt_len, tests[i].it_count, key, key_len);
-
-        printf("\ntest %i: ", i + 1);
-        printf("key %s", memcmp(tests[i].key, key, 16) ? "is bad" : "is good");
-        for(j = 0; j < key_len && j < 64; j += 4)
-        {
-            if(j % 16 == 0)
-                printf("\n");
-            printf("0x%02x%02x%02x%02x ", key[j], key[j + 1], key[j + 2], key[j + 3]);
-        }
-        printf(j < key_len ? " ... \n" : "\n");
-    }
-    printf("\n");
-    return 0;
-}
-
-#if defined(__cplusplus)

```

Comment:
Does this #if block need to actually be moved outside of the `TEST` section? I just noticed that it seems to add the closing brace for the `{` added at the very top of this file.